boot.sh

```diff
@@ -3,6 +3,9 @@
 cd /home/shippable/reqProc
 mkdir -p logs
 
+REQEXEC_DEST_DIR="$REQEXEC_DIR/src"
+cp -r $REQEXEC_SRC_DIR $REQEXEC_DEST_DIR
```

Comment:
Let's make use of the `REQ_EXEC_PATH` env set in the Dockerfile (change the env for consistency if you prefer to something appropriate). Let's copy this to `REQEXEC_SRC_DIR`.